packages/next/pnpm-lock.yaml

```diff
@@ -6,7 +6,7 @@ settings:
 
 dependencies:
   next:
-    specifier: ^13.2 || ^14
+    specifier: '>=13.2.0 <15'
```

Comment:
```suggestion
    specifier: '>=13.2.0'
```

Reply:
This package is depending on `localFont` from `next`. We can't guarantee whether there will be any breaking changes of it in `> 14`. I'm not sure whether removing the version cap here is a good idea.